app/authorisation.js

```diff
@@ -1,23 +1,21 @@
 var config = require('./config');
+const log = require('bole')('middleware');
 
 
-module.exports.denyUnauthorized = function (req, res, next) {
-
-  if (isAuthorized(req)) {
+module.exports = function (req, res, next) {
+  if (_isAuthorized(req)) {
     next();
-
   } else {
     res.sendStatus(403);
     next(new Error('User is not authorized'));
   }
 };
 
-function isAuthorized(req) {
+function _isAuthorized(req) {
   try {
     return req.user.nickname.toLowerCase() === config.rstudio.user;
-
   } catch (error) {
-    // do nothing
+    log.error('Error while checking user nickname: ', error)
```

Comment:
Missing semicolon

Reply:
Does this output anything useful? You might need to do `JSON.stringify(error)`

Reply:
I tested this yesterday by manually `throw` a weird json-y exception and `bole` escaped it correctly (which is great). I think log this is not a bad idea as this is kind of a programming error (e.g. the token doesn't include the `nickname` field).

Reply:
👍